hack/sast.sh

```diff
@@ -31,7 +38,7 @@ parse_flags "$@"
 echo "> Running gosec"
 gosec --version
 if [[ "$gosec_report" != "false" ]]; then
-  echo "Exporting report to $root_dir/gosec-report.sarif"
+  echo "Exporting report to $report_dir/gosec-report.sarif"
   gosec_report_parse_flags="-track-suppressions -fmt=sarif -out=gosec-report.sarif -stdout"
```

Comment:
Shouldn't the report dir be specified on this line, e.g.:
```suggestion
  gosec_report_parse_flags="-track-suppressions -fmt=sarif -out=$report_dir/gosec-report.sarif -stdout"
```